services/mac_manager/src/main/java/com/futurewei/alcor/macmanager/service/implement/MacRedisServiceImpl.java

```diff
@@ -59,6 +59,10 @@ public MacState getMacStateByMacAddress(String macAddress) {
 
     public MacState createMacState(MacState macState) throws Exception {
         MacAddress macAddress = new MacAddress();
+        if (macState.getState() == null)
+            macState.setState(MacUtil.MAC_STATE_ACTIVE);
```

Comment:
Assuming that MAC_STATE_ACTIVE is a constant, we should collect them and store in some constant file.

Reply:
I agree. Currently, all constants are in MacUtil class. But once we decide a location, I will put them there.